src/main/kotlin/frc/robot/subsystems/vision/VisionSubsystem.kt

```diff
@@ -0,0 +1,65 @@
+package frc.robot.subsystems.vision
+
+import edu.wpi.first.networktables.NetworkTableEntry
+import edu.wpi.first.networktables.NetworkTableInstance
+import org.ghrobotics.lib.commands.FalconSubsystem
+
+object VisionSubsystem : FalconSubsystem() {
+    val table = NetworkTableInstance.getDefault().getTable("limelight")
+
+    ///////////////////////
+    // Reading Functions //
+    ///////////////////////
+    fun getHasTargets(): Boolean {
```

Comment:
You could also use property access instead of function call, like this:
```kotlin
val hasTarget
    get() = (table.getEntry("tv").getBoolean(false)